examples/basic/main.tf

```diff
@@ -14,19 +14,18 @@ provider "aws" {
 module "expel_aws_cloudtrail_integration" {
   source = "../../"
 
-  expel_customer_organization_guid        = var.expel_customer_organization_guid
-  expel_assume_role_session_name          = "ExpelServiceAssumeRoleForCloudTrailAccess"
-  queue_message_retention_days            = 10
+  expel_customer_organization_guid = var.expel_customer_organization_guid
+  expel_assume_role_session_name   = "ExpelServiceAssumeRoleForCloudTrailAccess"
+  queue_message_retention_days     = 10
 
   enable_sqs_encryption                   = true
-  enable_cloudtrail_bucket_encryption     = true
   enable_cloudtrail_log_file_validation   = true
   enable_bucket_access_logging            = true
   enable_access_logging_bucket_encryption = true
   enable_bucket_versioning                = true
   enable_bucket_encryption_key_rotation   = true
 
-  prefix = "expel-cloudtrail-integration"
+  prefix = "expel-aws-integration"
```

Comment:
How about `expel-aws-cloudtrail-integration` ? We'll hopefully soon have `expel-aws-guardduty-integration` :-)

Reply:
there's a 63 chars limit on bucket name & with a uuid suffix of 36 chars, prefix limit is at 26 chars

Reply:
I think those are the three most important words -- LGTM!